spec/fixtures/education_benefits_claims/5490/kitchen_sink.json

```diff
@@ -5,7 +5,7 @@
   "outstandingFelony": true,
   "restorativeTraining": true,
   "vocationalTraining": true,
-  "trainingState": "DC",
+  "trainingState": "virginia",
```

Comment:
Since this field is passthrough, it doesn't matter too much, but is it the full state name or the postal abbreviations?